blog/docs/articles/getting-started-blog-p1.md

```diff
@@ -0,0 +1,262 @@
+
+Before you propose a new feature or report a bug, try __searching__ whatever platform the community uses to keep track of issues for a duplicate. If one already exists and has been 
+approved, great! You can move on and get started on the next step of the process. If it exists and __hasn't been approved__ yet, you can always __comment on the issue__ to indicate 
+that you have also experienced this bug, or want this feature. Finally, if it __does not exist__ yet then you should __propose__ the feature or __report__ the bug. Try and include
+as much detail in your proposal/bug report as possible, it will speed up the process. Regardless of whether your change was proposed before or you proposed it yourself, once it has
+been approved you can move on to the next step!
+
+### Coding your changes!
+
+![](/blog/images/getting-started-blog-series/post1/start-coding.png)
+
+Once your proposal has been accepted, it's time to code your change. In this section we aren't going to spend much time on discussing how to write the code itself (but if you're
+interested in how to write Knative code be sure to check out the rest of this blog series!). Instead, we will be focusing on all the processes you should follow around writing code.
+Generally, the process you will go through to code your changes is:
+
+1. Fork the project and clone it locally
+
+2. Make a branch
+
+3. Commit
+
+4. Create a PR
+
+5. Request reviews
+
+6. Make changes upon request
+
+7. PR is approved.
+
+8. Your code gets merged!
+
+Firstly, you should generally be working on a fork of the project. You can think of a fork as your own personal copy of the repository, where you have the permissions to make whatever
+changes you want. This is different from the main project repository, where you likely don't have permissions to change the code. To create a fork on GitHub, you can follow 
+[these instructions](https://docs.github.com/en/get-started/quickstart/fork-a-repo){:target="_blank"}. To create a fork on GitLab, you can follow [these instructions](https://docs.gitlab.com/ee/user/project/repository/forking_workflow.html){:target="_blank"}.
+Some important terminology here is that the repository you forked is generally referred to as the "upstream" repository, and we will be referring to your fork as the "origin",
+as that is normally how your local git remotes will be set up. To easily clone a forked repository and set up the git remotes properly, you can use the `git clonefork` command
+which can be installed from [here](https://github.com/cali0707/git-utils){:target="_blank"}. Alternatively, you can use the [git clone](https://git-scm.com/docs/git-clone){:target="_blank"} and 
+[git remote](https://git-scm.com/docs/git-remote){:target="_blank"} commands yourself to configure your local copy of your fork. Note that you will want to add a remote to the upstream repository,
+and that this remote is commonly named "upstream".
```

Comment:
`Note that you will want to add a remote to the upstream repository, and that this remote is commonly named "upstream".`
Not sure what you mean with that?

Reply:
I added a bit of explanation for this